EIPS/eip-2315.md

```diff
@@ -312,9 +316,7 @@ Bytecode: `0x6005565c5d5b60035e` (`PUSH1 0x05, JUMP, BEGINSUB, RETURNSUB, JUMPDE
 Consumed gas: `30`
 
 ## References
-Charles H.Moore, Geoffrey C. Leach. [FORTH - A Language for Interactive Computing](http://www.ultratechnology.com/4th_1970.pdf). 1970
-Philip J. Koopman, Jr. [Stack Computers: the new wave](https://users.ece.cmu.edu/~koopman/stack_computers/index.html). 1989
-Gavin Wood. [Ethereum:  A  Secure  Decentralized Generalized  Transaction  Ledger](https://ethereum.github.io/yellowpaper/paper.pdf). 2015-2021
+Gavin Wood. [Ethereum:  A  Secure  Decentralized Generalized  Transaction  Ledger](https://ethereum.github.io/yellowpaper/paper.pdf). 2014-2021
 Martin Lundfall.  [EIP-2327: BEGINDATA Opcode](https://github.com/ethereum/EIPs/blob/master/EIPS/eip-2327.md). 2019
 Nick Johnson. [EIP-3337: Frame pointer support for memory load and store operations](https://github.com/ethereum/EIPs/blob/master/EIPS/eip-3337.md). 2021
```

Comment:
This whole section needs to be removed.  This should not have been merged as Review with this section included.

Reply:
Agreed - Git accident.  They were removed a few commits back.